src/main/java/com/sqltest/demo/repository/SalesOrderRepo.java

```diff
@@ -2,6 +2,13 @@
 
 import com.sqltest.demo.model.SalesOrder;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
-public interface SalesOrderRepo extends JpaRepository<SalesOrder,Long> {
+import java.util.List;
+
+public interface SalesOrderRepo extends JpaRepository<SalesOrder, Long> {
+
+    // Add this method to find SalesOrders where bidding is "No"
```

Comment:
Aditya will also be using findByBidding where bidding will be YES. Remove the comment.